api/lookup.go

```diff
@@ -16,10 +15,11 @@ type Lookup struct {
 	Domain      string   `json:"domain" xml:"domain"`
 	MD5Hash     string   `json:"md5Hash" xml:"md5Hash"`
 	ValidFormat bool     `json:"validFormat" xml:"validFormat"`
-	Deliverable string    `json:"deliverable" xml:"deliverable"`
+	Deliverable bool    `json:"deliverable" xml:"deliverable"`
```

Comment:
The reason I had create a string here is there are some cases where we are not sure, such as ucla domains. The message might cover these cases though

Reply:
Do you want to keys here - boolean is simpler to test. Expanded message is also helpful?

Reply:
@lguegan I switched to boolean just for consistency and the ease of testing. But yes, I understand that a `true`/`false` could be misleading when trumail is unable to verify. Do you suggest that we stick to using strings?

@McManCSU Are you suggesting that we return just a boolean and a message instead of multiple keys?